app/assets/stylesheets/application.scss

```diff
@@ -12,4 +12,5 @@
  *
  */
 
-@import "bootstrap";
\ No newline at end of file
+@import "bootstrap";
+@import "margin_class_styles";
```

Comment:
Make this a partial file by just adding the `_` symbol, something like `_ margin_class_styles `  and just import it normally `@import "margin_class_styles";`